src/main/java/net/discordjug/javabot/listener/filter/MessageFormatFilter.java

```diff
@@ -0,0 +1,7 @@
+package net.discordjug.javabot.listener.filter;
+
+public interface MessageFormatFilter {
```

Comment:
This interface isn't really about formats. I think `MessageFilter` would be a better name.